django/db/backends/oracle/features.py

```diff
@@ -137,6 +137,25 @@ def django_test_skips(self):
                     },
                 }
             )
+        if self.connection.settings_dict["OPTIONS"].get("pool"):
+            skips.update(
+                {
+                    "Pooling does not supports persistent connection": {
```

Comment:
Typo: move the 's' from "supports" to "connection", i.e "Pooling does not support persistent connections"